documents/operating.md

```diff
@@ -13,10 +13,44 @@ Contents:
 ---------------------------------------------------------------------
 SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
 ---------------------------------------------------------------------
-- [ ] Company term
-- [ ] Continuance of company
-- [ ] Rickroll
-- [ ] Game of Mario Party 3 before legal arbitration between members
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
+### Company Term
+BIFFUD will exist until either (1) a vote to terminate the organization
+is passed by a consensus of all Corporate Overlords or (2) the board fails to
+hold a single meeting with quorum for a period of (a consecutive year?),
+(3) everyone dies / there are no more Corporate Overlords, or (4) if all
+Corporate Overlords are straight white cis men.
```

Comment:
In (1) we should define what "consensus" means unless Pennsylvania LLC law or some other part of the agreement provides a definition for it. Simple majority of COs? Majority based on CO ownership %? Tiebreakers?

(4) is awesome (but also gives the non-white, non-cis, non-men bloc a mutually assured destruction bargaining chip over the white cis men bloc. which is also kind of awesome.) One concern about it is that an amendment to the operating agreement could render the point moot (if, at some point in the future, white cis men are the majority of members).

Though [according to this](http://www.myllcoperatingagreement.com/Amending-your-llc-operating-agreement.htm) the default in Pennsylvania is that an operating agreement amendment requires the assent of _all_ members (not just a majority); even though, we should clear that with the lawyer OR explicitly write into the operating agreement that "Amendments to this operating agreement require the unanimous approval of *all* current Corporate Overlords").